lib/modals/failure.dart

```diff
@@ -0,0 +1,7 @@
+import 'package:flutter/material.dart';
+
+class Failure {
+  final String error;
```

Comment:
I see you are trying to use an approach from the web projects. It is not bad, but can we save only the error key and after we can create some errors map. In the future, it can ease i18n.

Reply:
Done